tests/unit/igraph_widest_paths.c

```diff
@@ -0,0 +1,28 @@
+/*
+   IGraph library.
+   Copyright (C) 2021  The igraph development team <igraph@igraph.org>
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include <igraph.h>
+
+#include "test_utilities.inc"
+
+int main() {
+
+    // TODO: Add tests
+
+    return 0;
```

Comment:
Make sure to fail the test by returning 1, or to skip the test by returning 77 (magic exit code) so we don't accidentally merge this PR without writing tests.

Reply:
Done, used exit code 77